float64 kp_in
float64 kd_in
---
bool result